Resources/public/less/website-toolbar.less

```diff
@@ -1,6 +1,11 @@
 #ezwt-content input {
-    vertical-align: baseline;
+  vertical-align: baseline;
 }
+
 #ezwt-content input[disabled] {
-    background-color: transparent;
+  background-color: transparent;
 }
+
+#ezwt a img {
```

Comment:
@clash82 / @lserwatka I think we removed ezwt recently, @lolautruche might be able to confirm otherwise check the legacy removal PR that was merged. (ezwt is purely legacy)